### ISSUE TITLE
Fibaro wall plug customisation 

### ISSUE THREAD

**Issue by prbreezy** [damianxd/homebridge-vera#131]
Hi 

I have this fibaro wall plug https://www.fibaro.com/en/products/wall-plug/ it's controlling a fan currently it's show as a light there is anyway to change the type to fan to have a fan icon.

Cheers.

**Comment by neptunix** [comment]
Change subcategory_num for this device from 0 to 4 (at vera web interface -> Device -> Advanced) and restart homebridge

**Comment by prbreezy** [comment]
Now it's looks like a smoke sensor it was 1 and change to 4, I will try 3 and 2.

**Comment by neptunix** [comment]
Are you happy with your wall plug btw? Does it support power metering in Vera UI?

**Comment by prbreezy** [comment]
Yes very happy ! It's power metering the power and it's quiet good because this wall plug is very tight and fit everywhere.

**Comment by prbreezy** [comment]
Here a screenshot https://ibb.co/c4N9u5

**Comment by neptunix** [comment]
Great to know that!
